cloud-etl/config/demo.cfg

```diff
@@ -54,20 +54,22 @@ export DESTINATION_STORAGE='s3'
 # S3_BUCKET: bucket name
 # Demo will modify contents of the bucket
 # Do not specify one that you do not want accidentally deleted
-#export S3_BUCKET='cloud-etl-example'
+# Make sure to change the name of this bucket
+#export S3_BUCKET='change-me'
 
 #---------------------------------------------------------
 # GCP GCS
 #---------------------------------------------------------
 
 # GCS_CREDENTIALS_FILE: json-formatted file with credentials, downloaded from the Google Cloud Console
 # File should resemble example in https://cloud.google.com/iam/docs/creating-managing-service-account-keys#creating_service_account_keys
-#export GCS_CREDENTIALS_FILE=~/.config/gcloud/cloud-private.json
+#export GCS_CREDENTIALS_FILE=~/.config/gcloud/confluent-etl-demo-keys.json
+#export GCS_PROJECT_ID='devx-testing'
```

Comment:
change to something generic like `your-project-id`?

Add `GCS_PROJECT_ID` to `index.rst` (add to #5 in the [setup](https://docs.confluent.io/cloud/current/get-started/examples/cloud-etl/docs/index.html#setup) section)

Reply:
ack